src/cal/atmcorr_specmath.py

```diff
@@ -178,8 +201,20 @@ def main():
         rad_file = ''
         xml_file = ''
 
+        """
+
+        #UNCOMMENT THIS BLOCK AND REMOVE CHANGE:
+        #folder_dir = working_dir
+        #IN ORDER TO SEARCH THROUGH THE SUBDIRECTORIES OF THE INPUTTED
+        #DIRECTORY
+        
         # Saves the directory of the folder
         folder_dir = os.path.join(working_dir, folder)
+        """
```

Comment:
Generally in code comments do not go under `"""` I know it is convenient, but it is better if `#` are used. I, for example, add as many `#` as needed.